packages/ckeditor5-table/docs/_snippets/features/table-caption.html

```diff
@@ -0,0 +1,32 @@
+<div id="snippet-table-caption">
+	<figure class="table">
+		<table>
+			<tbody>
+				<tr>
+					<td>
+						<p style="text-align:center;">Friendly</p>
+					</td>
+				</tr>
+				<tr>
+					<td>
+						<p style="text-align:center;">Intelligent</p>
+					</td>
+				</tr>
+				<tr>
+					<td>
+						<p style="text-align:center;">Generally good with people</p>
+					</td>
+				</tr>
+				<tr>
+					<td>
+						<p style="text-align:center;">Has a lot of energy</p>
+					</td>
+				</tr>
+			</tbody>
+		</table>
```

Comment:
Maybe some table with more than one column would make more sense?

Reply:
I spent more time trying to find some general data example than I want to admit ;) Adding another column could look nicer, but it's irrelevant to the feature it presents - a table caption. Saying that, I am more than happy to update it if you have better data example to present it here.

Reply:
Maybe some simpler version of what we have in the table manual test? http://fake.ckeditor.com:8125/ckeditor5-table/tests/manual/table.html